third_party/move/move-compiler/transactional-tests/tests/constants/constant_break.move

```diff
@@ -0,0 +1,8 @@
+//# run
+script {
```

Comment:
No need for a transactional test, as it does not pass type checking.

Reply:
Ok.  I was just being paranoid.